### DISCUSSION TITLE
Mấy con cá vàng giúp mình bảo vệ hệ thống thế nào?

### DISCUSSION THREAD

**Discussion by giscus** [thangchiba/ThangChibaBlog#4] [Announcements]
# Mấy con cá vàng giúp mình bảo vệ hệ thống thế nào?

Bạn có thắc mắc máy tính có thể thực sự tạo ra được số ngẫu nhiên không? Làm sao mấy con cá vàng vô tri lại có thể giúp mình tạo được mã ngẫu nhiên để chống bị hack quy luật tạo mã? Đây cũng là cách mà CloudFlare đang làm để bảo vệ một phần lớn traffic internet trên thế giới này.

https://192.168.1.74:3435/vi/blog/random-generator/

**Comment by thangchiba** [comment]
Cá vàng
